frontend/src/i18n/locales/zh-cn.json

```diff
@@ -69,8 +69,10 @@
     "fee": "Umbra 费",
     "fee-explain": "在 %{chainName} 上交易非常便宜，因此会收取少量费用以阻止垃圾信息。",
     "total": "总额",
+    "max": "最大限度",
```

Comment:
@garyghayrat think I forgot to ask you to confirm this translation

Reply:
```suggestion
    "max": "最大限额",
```